### ISSUE TITLE
Update Pod Spec to work with TVOS 

### ISSUE THREAD

**Issue by dzeitman** [lukeredpath/libPusher#194]
Currently you can't use libPusher in a TVOS application. 

Here's info on how to update Pod to handle TVOS.
http://blog.cocoapods.org/CocoaPods-0.39/

**Comment by hamchapman** [comment]
There's a pull request for that here: https://github.com/lukeredpath/libPusher/pull/190

Looks like it can now be merged

**Comment by dzeitman** [comment]
- New to this...  I updated cocoa pods  -- 
Now get this error:
[!] The platform of the target `APPNAME` (tvOS 9.0) is not compatible with `libPusher (1.6.1)`, which does not support `tvos`.

**Comment by hamchapman** [comment]
That'll be because the pull request on libPusher isn't merged yet. If you want to try it out before it gets merged then add this to your Podfile:

```
pod 'libPusher', git: 'https://github.com/lukeredpath/libPusher.git', branch: 'add-tvos-support'
```

**Comment by dzeitman** [comment]
Thanks!

There's quite a few pod libs out there that would be useful for TVOS.
 I'm still new to cocoa pods and git.   (rather I've been asleep)

What are the steps one would need to update other pod libs to work with
tvos ?

Dan

**Comment by goose2460** [comment]
I'm getting 'The platform of the target `Pods` (tvOS 9.1) is not compatible with `SocketRocket (0.4.1)`, which does not support `tvos`.' when I try the Podfile @hamchapman suggested

**Comment by hamchapman** [comment]
Yeah, at the moment SocketRocket doesn't support tvOS (https://github.com/square/SocketRocket/pull/279).

I'll try and fix up that pull request and get the branch building on CI for SocketRocket and then we can update the dependency for libPusher and get it supported.

**Comment by hamchapman** [comment]
#190 will fix this, although relevant to that is #219
